src/search/pdbs/pattern_collection_generator_combo.cc

```diff
@@ -29,15 +32,16 @@ PatternCollectionInformation PatternCollectionGeneratorCombo::generate(
     TaskProxy task_proxy(*task);
     shared_ptr<PatternCollection> patterns = make_shared<PatternCollection>();
 
-    PatternGeneratorGreedy large_pattern_generator(max_states);
-    const Pattern &large_pattern = large_pattern_generator.generate(task).get_pattern();
-    patterns->push_back(large_pattern);
-
+    PatternGeneratorGreedy large_pattern_generator(max_states, rng);
+    Pattern large_pattern = large_pattern_generator.generate(task).get_pattern();
     set<int> used_vars(large_pattern.begin(), large_pattern.end());
+    patterns->emplace_back(move(large_pattern));
```

Comment:
I think we generally use push_back() in cases like this (and emplace_back() only if the object is constructed, not just moved).